README.md

```diff
@@ -121,7 +121,7 @@ If you require the absolute latest version of `ng-faker` the `master` branch @ h
 
 | [<img src="https://github.com/rotimi-babalola.png" width="100px;"><br><sub><b>Rotimi Babalola</b></sub>](https://github.com/rotimi-babalola)| [<img src="https://github.com/BolajiOlajide.png" width="100px;"><br><sub><b>Bolaji Olajide</b></sub>](https://github.com/BolajiOlajide) | [<img src="https://github.com/oyinda-subair.png" width="100px;"><br><sub><b>Subair Oyindamola</b></sub>](https://github.com/oyinda-subair)
 | :---: | :---: | :---: | 
-| [<img src="https://github.com/olamilekan000.png" width="100px;"><br><sub><b>Olamilekan Odukoya</b></sub>](https://github.com/olamilekan000) |
+| [<img src="https://github.com/olamilekan000.png" width="100px;"><br><sub><b>Olamilekan Odukoya</b></sub>](https://github.com/olamilekan000) | [<img src="https://github.com/Ayomiga.png" width="100px;"><br><sub><b>Ayomiga</b></sub>](https://github.com/Ayomiga) |
```

Comment:
Can you go through the contributor list [here](https://github.com/BolajiOlajide/ng-faker/graphs/contributors) and add all the contributors here?

That'd be a really great thing to have. Thanks @Ayomiga

Reply:
Never mind.